modules/specialized-guides/pages/public-cloud-guide/payg/payg-faq.adoc

```diff
@@ -0,0 +1,254 @@
+
+**I have purchased multiple SUSE products from the AWS Marketplace (i.e. Rancher, NeuVector and SUSE Manager), does the AWS Marketplace billing method still work?**:: 
+**Yes**.  The billing mechanism for the various products is independent.  Each deployment will be billed separately via the AWS Marketplace. 
+
+**I already have an existing SUSE Manager deployment, how can I get this deployment to bill via the AWS Marketplace?**:: 
+Existing deployments cannot be directed converted or updated to provide billing via the AWS Marketplace. 
++
+
+It is easy to migrate your existing SUSE Manager deployment to a SUSE Manager instance which has been deployed from and can bill via the AWS Marketplace.
+
+
+== Technical (Product)
+
+**Can I migrate an existing SUSE Manager instance to a PAYG version?**:: 
+**Yes**, it is possible to use a migration script to migrate the SUSE Manager database to a new PAYG instance of SUSE Manager from the AWS Marketplace.  This has the benefit of maintaining event history for managed instanced. Salt states and synchronized packages are also moved as part of this process. 
++
+
+If event history does not need to be retained as part of the move, customers can simply deploy a new SUSE Manager instance from the AWS Marketplace and re-onboard all clients. 
+
+// Add link to docs on this process
+ 
+**How do I obtain support?**:: 
+It is very simple to open a support case with SUSE for SUSE Manager.   
++
+
+Create a ‘supportconfig’ via SUSE Manager instance CLI and upload the output to the SUSE Customer Center at the following  
++
+
+link:https://scc.suse.com/cloudsupport[]
++
+
+Providing the billing is active, a support case will be opened. 
+
+**Is there any difference between the SUSE Manager product on the AWS Marketplace compared to the versions I can run in my own data center?**:: 
+The SUSE Manager product available in the AWS Marketplace is the same product, with the same functionality that you would run on-premises or with a manual installation.   
++
+
+The only exception being the ability to bill via the AWS Marketplace and connect to the SUSE Update Infrastructure for SUSE Manager product updates. 
+
+**Does the SUSE Manager instance need to run 24/7?**:: 
+To ensure continuity with support it is recommended that the SUSE Manager instance always remains active.   
+
+**How do I get fixes and updates for SUSE Manager, do I need a subscription key?**:: 
+SUSE Manager, when deployed via the AWS Marketplace listing (monthly billing) will automatically connect to the SUSE Update Infrastructure in AWS, no separate SUSE Manager Subscription Key is required.  
++
+
+Updates to the SUSE Manager instance can be applied from those channels using standard SUSE tooling (i.e. zypper). 
+
+**Is the use of SUSE Manager Hub supported with this listing?**:: 
+**Yes**.  SUSE Manager Hub is recommended for customer with more than 10,000 managed clients.
```

Comment:
@admd @rjmateus - SUMA Hub in the cloud via PAYG? Did we ever talk about it? I do not see why it should not work, but better to ask.